### PR TITLE
Drop rhel-8-7 and rhel-9-1

### PR DESCRIPTION
**Pull request by martinpitt**
I checked with @ptoscano that this is okay for sub-man.

### CODE REVIEW

File: lib/testmap.py [hunk 1]
```diff
@@ -189,16 +181,13 @@
...
-            'rhel-9-1',
             'centos-9-stream',
             'rhel-9-2',
             'fedora-36',
             'fedora-37',
         ],
         '_manual': [
             'centos-8-stream/subscription-manager-1.28',
-            'rhel-8-7',
-            'rhel-8-7/subscription-manager-1.28',
```
Comment (ptoscano) [inline lib/testmap.py:201]: can you please move the testing using the sub-man-1.28 branch to 8.9?

**Comment by ptoscano** [comment]
Left only one comment, the rest LGTM.

File: lib/testmap.py (hunk 1 above)
Comment (martinpitt) [reply lib/testmap.py:201]: You mean for regularly testing it, instead of only `_manual`? As it already is on `_manual`, and so is this removal. But sure, let me check whether this works -- I triggered "rhel-8-9@candlepin/subscription-manager/subscription-manager-1.28" here. This PR needs work anyway, due to [this](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4659-20230417-045930-0a83a092-rhel4edge-cockpit-project-cockpit/log.html).

Comment (ptoscano) [reply lib/testmap.py:201]: `_manual`-only is fine, yes

Comment (martinpitt) [reply lib/testmap.py:201]: [works](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4659-20230417-050759-0a83a092-rhel-8-9-candlepin-subscription-manager-subscription-manager-1.28/log.html), added!

Comment (martinpitt) [reply lib/testmap.py:203]: It already was on manual. But I suppose it doesn't hurt to regularly test it on 8-9?

File: lib/testmap.py [hunk 2]
```diff
@@ -236,7 +225,7 @@
 # their non-Atomic siblings.
 OSTREE_BUILD_IMAGE = {
     "fedora-coreos": "fedora-37",
-    "rhel4edge": "rhel-9-1",
+    "rhel4edge": "rhel-9-0",
```
Comment (KKoukiou) [inline lib/testmap.py:228]: Is this desired that rhel4edge is updated down and not up?

Comment (martinpitt) [reply lib/testmap.py:228]: Yes, see the commit message. I tried to bump it to 9.2, but as the rhel4edge image itself is still 9.1, a 9.2 build [fails to install](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4659-20230417-045930-0a83a092-rhel4edge-cockpit-project-cockpit/log.html). Upwards compatibility is not a problem.

File: lib/testmap.py (hunk 1 above)
Comment (ptoscano) [reply lib/testmap.py:203]: Sorry, I was not clear: what I meant was replacing `rhel-8-7/subscription-manager-1.28` (ie with the old sub-man branch) with `rhel-8-9/subscription-manager-1.28`, still in `_manual`; there is already `rhel-8-9` (ie with sub-man/main) in `_manual`

Comment (ptoscano) [reply lib/testmap.py:203]: ... thinking about it further, considering there is already `centos-8-stream/subscription-manager-1.28` there, it should be OK for now (and new combinations can always be tested locally anyway)

tl;dr: changes are fine, thanks!

**Review by ptoscano — DISMISSED** [review]

**Review by KKoukiou — DISMISSED** [review]

File: lib/testmap.py (hunk 1 above)
Comment (martinpitt) [reply lib/testmap.py:203]: Now I understand -- you mean "rhel-8-7/subscription-manager-1.28" is a scenario actually understood by sub-man. Yes, I'll do that then. Updated!

**Review by ptoscano — APPROVED** [review]

**Review by KKoukiou — APPROVED** [review]
